ducktape/command_line/main.py

```diff
@@ -141,6 +157,10 @@ def main():
             print "parameters are not valid json: " + str(e.message)
             sys.exit(1)
 
+    if args.report_only:
+        generate_reports(args.session_id, args.results_root)
```

Comment:
`--session-id` isn't required, you might need to check that it was set / is valid.

Also, it doesn't look like this matches the argument list. I think you're missing the `result_store`, which isn't generated until later in this function.